docs/one-click-integrations/github-integration.md

```diff
@@ -112,6 +117,14 @@ The Pulse GitHub integration detects incidents automatically using [pull request
 -   The incident creation date is the timestamp when the reverted pull request was initially merged. If Pulse can't get the reverted pull request number from the branch name, the incident creation date is the timestamp of the first commit to the incident pull request.
 -   Pulse associates incidents to the system matching the repository name.
 
+### Use pull request naming convention (based on default branch) {: id="gh-incident-naming"}
+
+-   Pulse executes the same incident detection mechanism based on [pull request reverts](#gh-incident-pr-revert) using the naming convention you define instead of the default branch prefix `revert-`.
+
+-   You can define the prefix for the branch name or the pull request title to match the naming convention on your workflow.
```

Comment:
I believe we can use `of` here since this could be reworded as `your workflow's naming convention`:

```suggestion
-   You can define the prefix for the branch name or the pull request title to match the naming convention of your workflow.
```